packages/ckeditor5-typing/tests/input.js

```diff
@@ -514,6 +514,43 @@ describe( 'Input feature', () => {
 			expect( getViewData( view ) ).to.equal( '<p><strong>Foo</strong> {}</p>' );
 		} );
 
+		it( 'should handle bogus br correctly (with custom br to softBreak converter)', () => {
```

Comment:
How did you come up with these tests? Did you encounter some regressions after making this change? Does it anyhow affect beforeinput-related work?

Reply:
The case is that those bogus BRs are stripped while converting to the model (input compares the current model and the model generated from the modified DOM). This case won't affect the beforeinput because it's not used there anymore.